BuildAndTest.proj

```diff
@@ -79,17 +79,23 @@
           SkipUnchangedFiles="true"
           UseHardlinksIfPossible="true" />
 
-    <MakeDir Directories="$(CoreClrTestDirectory)\xUnitResults" />
+    <MakeDir Directories="$(CoreClrTestDirectory)/xUnitResults" />
+
+    <!-- Publish binaries for CoreClr test -->
+    <MSBuild BuildInParallel="true"
+             Projects="src/Test/DeployCoreClrTestRuntime/DeployCoreClrTestRuntime.csproj"
```

Comment:
I'm happy with the pedantry although I still strongly believe the slashes should go the other way.